mbed_lib.json

```diff
@@ -18,9 +18,9 @@
             "value": "DEFAULT_MAX_APPLICATION_SIZE"
         },
         "trace": {
-            "help": "Show boot status and progress on serial output. Disable output to save space on headless devices.",
-            "value": true,
-            "macro_name": "MBED_BOOTLOADER_TRACE"
+            "help": "Show boot status and progress on serial output. Use USE_DIRECT_SERIAL_OUTPUT to save space.",
```

Comment:
maybe better to add to all targets
we are on the edge at least for disco , adding fw key support - disco gets bigger 32k

Reply:
this is for all targets, as it is located in `mbed_lib.json`